resources/assets/less/bem/osu-layout.less

```diff
@@ -166,9 +165,75 @@
       ._page;
       padding: 20px;
       margin-bottom: 10px;
+    }
+
+    &--page-admin {
+      ._page;
+      padding: 20px;
+      margin-bottom: 10px;
+      color: white;
+
+      h1, h2, h3, h4 {
+        font-style: normal;
+        color: white;
+      }
+
+      h1 {
+        font-size: 28px;
+      }
+
+      h2 {
+        font-size: 16px;
+        display: inline-block;
```

Comment:
this probably should be `width: max-content` instead so it'll still occupy the entire row instead of becoming inlined element.

Reply:
and a `max-width: 100%` :dancer: